src/stac_iitmandi/templates/stac_iitmandi/_navbar.html

```diff
@@ -1,27 +1,40 @@
-
+{% load static %}
 <nav class="navbar navbar-expand-lg sticky-top bg-custom navbar-light mt-5">
-    <a class="navbar-brand" href="#">STAC</a>
-    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+<!--<nav class="navbar navbar-expand-lg navbar-dark bg-primary">-->
+    <a class="navbar-brand" href={% url 'stac-home' %}><img src="{% static 'stac_iitmandi/images/STAClogo.png' %}" class="img-fluid width:1%" alt="STAC"></a>
+    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#main_nav" aria-expanded="false" aria-label="Toggle navigation">
       <span class="navbar-toggler-icon"></span>
     </button>
-    <div class="collapse navbar-collapse" id="navbarSupportedContent">
-      <ul class="navbar-nav ml-auto">
-        <li class="nav-item active">
-          <a class="nav-link" href="#">HOME <span class="sr-only">(current)</span></a>
-        </li>
-        <li class="nav-item">
-            <a class="nav-link" href="#">ASTRAX</a>
-        </li>
-        <li class="nav-item">
-          <a class="nav-link" href="#">ZENITH</a>
-        </li>
-        <li class="nav-item">
-          <a class="nav-link" href="#">BLOG</a>
-        </li>
-        <li class="nav-item">
-            <a class="nav-link" href="#">ABOUT</a>
-        </li>
+    <div class="collapse navbar-collapse" id="main_nav">
```

Comment:
I think the `id` is not supposed to be changed from `navbarSupportedContent`.

Reply:
But it is of no use until now.

Reply:
> But it is of no use until now.

Then you should leave it as it is.